docs/api-specs.md

```diff
@@ -811,6 +815,10 @@ are not necessarily priced in Bitcoin.
 This endpoint is used to get the price of a name, denoted in a specific
 cryptocurrency (not necessarily Bitcoin).
 
+The `amount` given will be in the smallest possible units of the currency.  For
+`BTC`, the `amount` will be in satoshis.  For `STACKS`, the `amount` will be in
+micro-Stacks (1/10,000th of a Stack).
```

Comment:
A microstack is `1e-6`th of a Stack, right ? That's (1/1,000,000th).

Reply:
One hundred-thousandth.  We both done goofed.  1/100,000th

Reply:
I always get confused by prefixes and have to check like 10 times.
1e-6 is one one-millionth (1 / 10^6) and matches the SI micro prefix (https://en.wikipedia.org/wiki/Micro-) and the blockstack-core constant (https://github.com/blockstack/blockstack-core/blob/master/blockstack/lib/config.py#L98).

Reply:
yup, 1 millionth it is.